hyperspy/tests/io/test_nexus_hdf.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2020 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import os.path
+from os import remove
+import gc
+import tempfile
+import numpy as np
+import pytest
+from hyperspy.io import load
+import hyperspy.api as hs
+from hyperspy.signal import BaseSignal
+
+
+dirpath = os.path.dirname(__file__)
+
+file1 = os.path.join(dirpath, 'nexus_files', 'file1.nxs')
```

Comment:
The test files are too large: it should be possible to reduce the size of the file to almost nothing by filling with zeros and saving it with compression.

Reply:
Replaced with smaller files